app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="home-section">
+  <div class="home-nav">
+    <%= link_to "Budgetify", expenses_path, class: 'logo' %>
+    <div class="page-name">Expenses</div>
```

Comment:
- [ ] @Al-Amin-Khan-Shakil Great work implementing the changes requested, however, you still have not changed the page titles as previously requested, On this page instead of `Expenses` it should be  `CATEGORIES`, kindly consider implementing these changes requested.